dev-docs/bidders/pulsepointlite.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: PulsePoint Lite
+description: Prebid PulsePoint Lite Bidder Adaptor
+
+top_nav_section: dev_docs
+nav_section: reference
+
+hide: true
+
+biddercode: pulsepointLite
```

Comment:
code is actually `pulseLite`

Reply:
Thanks @mkendall07 . Yes, please use ```pulseLite``` instead. ```pulsepointLite``` had key truncation issues when sending params to DFP (character limit is 20 for keys in DFP).

Reply:
@aneuway2 ^^